content/general/concepts/vim/vim.md

```diff
@@ -0,0 +1,35 @@
+---
+Title: 'Vim'
+Description: 'Vim is a free, open source, and customizable text editor for Linux.'
+Subjects:
+  - 'Bash/Shell'
+  - 'Developer Tools'
+Tags:
+  - 'Vim'
+  - 'Linux'
+  - 'Developer Tools'
+CatalogContent:
+  - 'paths/computer-science'
```

Comment:
```suggestion
  - 'learn-the-command-line'
  - 'paths/computer-science'
```

The `CatalogContent` should always include two links:

 - One free course slug.
 - One paid `paths/` slug.